backend/contributions/views.py

```
@@ -3115,6 +3183,100 @@ def propose(self, request, pk=None):
             status=status.HTTP_200_OK
         )
 
+    @action(detail=True, methods=['post'], url_path='question-proposal')
+    @transaction.atomic
+    def question_proposal(self, request, pk=None):
```

Ruff (0.15.20)

[warning] Too many return statements (7 > 6)

(PLR0911)